### PR TITLE
Update privacy policy

### PR DESCRIPTION
**Pull request by sebgrebe**
- As discussed in weekly meeting on 4/7/2021  
- @victor-yunenko if you could please have a last look  
- If a native speaker could proof read, that would be great. Maybe @LisaJD or @walambert?

### CODE REVIEW

**Review by walambert — COMMENTED** [review]
Recommendations:

Line 38, rewrite suggestion: "As specified by law [specify which law either in parentheses or "As required by "law name" of "government / region which enforces that law", do you mean the GDPR? or something else?; is this British law only, EU law, international policy or law, or some combination thereof or something else?], we are the controller [maybe link to what that means? https://www.gdprsummary.com/gdpr-definitions/controller/] of the personal information received by us from you when you interact with us."

Line 45, rewrite suggestion: "To participate in our community, you must create a profile on the EA Hub, and in doing so you are required to provide us with the following information:"

Line 56, suggestion: embed a link to a page that gives an overview of what a giving pledge is and means

Line 82, rewrite suggestion: "We do so to improve the overall quality and effectiveness of the EA Hub, e.g. we may use such data to measure the efficacy of our marketing campaigns."

Line 86, grammar consistency: add a comma after sexual orientation, e.g. "...sexual orientation, or political opinions." The oxford comma (https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/) seems to be used consistently as a style throughout this page, so this change would bring this sentence in line with the overall style.

Line 104, rewrite suggestion: "We only use your personal information as permitted by law. Please see the following permitted actions and circumstances we commonly take or operate within regarding your personal information:

Line 118, specificy: mention which data protection law and link to it. (this might be a good general rule for the whole page)

Line 134, rewrite suggestion: remove "...a variety of...", e.g. "We employ physical and technical measures to keep..."

Line 142, rewrite suggestion: "To see our specific security measures, please view our Data Protection Policy." (with link included of course)

Line 148, question: are there explicitly written policies regarding how such information is stored and how access is granted and what roles or people have what access? If so, that'd be good to link to here; if not, that'd be good to write and then link to here.

Line 161, question: what policy guides the handling of personal information in the circumstances mentioned herein? 

Line 180, question: where if anywhere is a full listing of those rights written out? if such a listing exists, it'd be good to link to it here.

Cheers,
Willa

**Comment by sebgrebe** [comment]
Thanks, @walambert! Implemented most of them.

For future reviews: You can make comments on lines in the files you're reviewing themselves, so you don't need to mention the lines

File: eahub/templates/eahub/privacy_policy.html [hunk 1]
```diff
@@ -11,64 +11,181 @@
...
+<h3>Newsletter subscriptions</h3>
+<p>If you sign up to one of our monthly newsletters, or register for an event we run, we will disclose your name and email address to Mailchimp, a service we use to manage the process of sending mass emails. You can opt out of a mailing list at any time by clicking the appropriate link in any email we send through Mailchimp.</p>
 
-<h3>Data Retention</h3>
-<p>When you enter information to the Site, we will maintain your Account Information and make it publicly available until you delete this information yourself or ask us to delete it for you.</p>
+<h3>Other disclosures</h3>
+<p>If we choose to sell, transfer, or merge parts of the company or our assets, we may disclose your personal information to the new owners. Alternatively, we may seek to merge with another company. If a change like that happens, then the new owners may use your personal information in the same way as set out in this policy.</p>
```
Comment (viktor-yunenko) [inline eahub/templates/eahub/privacy_policy.html]: The only thing that I'm not sure about is this paragraph. Above we say that we will not sell the data, but this statement means the opposite?

**Review by viktor-yunenko — CHANGES_REQUESTED** [review]

**Comment by walambert** [comment]
> Thanks, @walambert! Implemented most of them.
> 
> For future reviews: You can make comments on lines in the files you're reviewing themselves, so you don't need to mention the lines

Ah great! I will do that next time.

File: eahub/templates/eahub/privacy_policy.html (hunk 1 above)
Comment (sebgrebe) [reply eahub/templates/eahub/privacy_policy.html]: There is a difference between selling the data and selling the company. This is to cover cases where the company gets new owners or becomes part of another organisation. For this new entity to be able to run the Hub, they need access to the data

Comment (viktor-yunenko) [reply eahub/templates/eahub/privacy_policy.html]: I guess it's fine for me, but shouldn't we remove the "will not sell the data" sentence then? I don't really see a distinction between selling the company and selling the data from the users point of view?

Comment (sebgrebe) [reply eahub/templates/eahub/privacy_policy.html]: The difference is that if the company gets sold, the new owners presumably would continue running the Hub, so from the users' perspective not much changes and it's actually in their interest, as they want the Hub to continue to exist. However, if we just sell the data to someone else, this is not in the users' interest as it doesn't support continue running the Hub.

Also, my default assumption here is that if it's fine for CEA (they have this verbatim in their policy, too), it should be fine for us.

**Review by viktor-yunenko — APPROVED** [review]
All looks good, thank you!

**Pull request merged**